workers.py

```diff
@@ -45,13 +45,19 @@ def _retryable(method: str, *args, **kwargs) -> requests.Response:
     raise requests.HTTPError('All attempts failed')
 
 
-def download_job(source_url: str, source_id: str, dest_url: str) -> None:
+def download_job(
+        source_url: str,
+        source_id: str,
+        dest_url: str,
+        b64_identity: str
```

Comment:
I would personally prefer this to be set to `None` as a default value

Reply:
right, `None` is great here!

Reply:
@AparnaKarve I think my English was not the best: I've meant to set a default value of the parameter to `None`, not state that the type should be `None`.

 ```suggestion
        b64_identity: str = None
```
^ that is what I've meant :wink: Sorry!

Reply:
Of course I understood what you meant...so my fault entirely -- sorry! :)

Reply:
...and addressed! :)